OrbitClientGgp/ClientGgp.cpp

```diff
@@ -76,10 +76,12 @@ bool ClientGgp::RequestStartCapture(ThreadPool* thread_pool) {
     return false;
   }
 
+  // Clean capture data and variables to start from a clean state
```

Comment:
I would remove this comment. It doesn't really help understanding this better. ClearCapture() is a pretty clear name.

Reply:
You are right, I have removed it, thanks!